helix-view/src/input.rs

```diff
@@ -230,6 +230,16 @@ impl From<crossterm::event::KeyEvent> for KeyEvent {
     }
 }
 
+#[cfg(feature = "term")]
+impl From<KeyEvent> for crossterm::event::KeyEvent {
+    fn from(KeyEvent { code, modifiers }: KeyEvent) -> Self {
+        crossterm::event::KeyEvent {
+            code: code.into(),
+            modifiers: modifiers.into(),
+        }
+    }
+}
```

Comment:
Needs to handle `shift-tab` to `backtab` with the shift modifier removed, basically reverse of https://github.com/helix-editor/helix/commit/cab09093dd5bf9b4707bfdfd8529b348c02670ea#diff-24a44e1d9b80513ef01be1d5d5bb1c5d99bd0b9dd9f55d633b2765157e93cc9bR219